Resources/Prototypes/_White/Entities/Objects/Misc/subdermal_implants.yml

```
@@ -0,0 +1,11 @@
+- type: entity
+  parent: BaseSubdermalImplant
+  id: NeuroStabilizationImplant
+  name: neuro stabilization implant
+  description: Blocks all incoming stamina damage at a cost of shock.
+  noSpawn: true
+  components:
+  - type: SubdermalImplant
+  - type: Tag
+    tags:
+    - NeuroStabilization
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)